core/modules/wpublish/ChunkInventory.h

```diff
@@ -79,6 +83,9 @@ class ChunkInventory {
     bool has(std::string const& db, int chunk,
              std::string table=std::string()) const;
```

Comment:
I don't think that our code calls this method with non-empty `table`, we should drop this parameter entirely and forget about per-table chunks.

Reply:
Aha! That parameter was actually another reason why I chose to inject column 'table' into the new table schema (qservw_worker.Chunks). I just forgot about it. Let me search through the code to see if what you say is true, and if I won't find anything then I will eliminate this parameter along with the column 'table' in the table schema. I will also simplify the corresponding transient collection of chunks within ChunkInventory.